src/components/organisms/PageHeader/PageHeader.styled.tsx

```diff
@@ -93,6 +97,7 @@ export const Logo = styled.img`
 
 export const LogoContainer = styled.div`
   height: 100%;
+  min-width: 50px;
```

Comment:
Why do we need a min-width if the width is already fixed? 🤔

Reply:
Because the container flex box pushes the logo back to 40px on small screens